### DISCUSSION TITLE
Django Or Node js

### DISCUSSION THREAD

**Discussion by SuBroXz11** [community/community#140172] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Which one is better for REST API?

**Comment by bishwozz** [comment]
look at here https://dev.to/matinmollapur0101/django-vs-nodejs-a-comparative-guide-for-full-stack-development-2ngd

**Comment by ebndev** [comment]
Hi @SuBroXz11 & @bishwozz, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
